syngenta_digital_dta/s3/adapter.py

```diff
@@ -238,3 +251,20 @@ def __set_acl(self, public_read):
 
     def __generate_publish_data(self, **kwargs):
         return {'presigned_url': self.create_presigned_read_url(**kwargs)}
+
+    def __create_tags(self, **kwargs):
+        tags = kwargs.get('tags', None)
+        result_tags = []
+        for key, value in tags.items():
```

Comment:
this will fail is tags kwarg is not provided, because None doesnt have .items method.

Reply:
updated